source/config/configSpec.py

```diff
@@ -54,6 +54,8 @@
 	WASAPI = featureFlag(optionsEnum="BoolFlag", behaviorOfDefault="enabled")
 	soundVolumeFollowsVoice = boolean(default=false)
 	soundVolume = integer(default=100, min=0, max=100)
+	soundSplitState = integer(default=0)
+	soundSplitToggleMode= integer(default=0)

```

Comment:
Maybe it would be worth using a list, as done with `excludedSpeechModes`. It would be more easy to add new modes if needed in the future, e.g. https://github.com/nvaccess/nvda/pull/16071#issuecomment-1902237001.